filament/backend/src/vulkan/VulkanContext.cpp

```diff
@@ -73,7 +73,7 @@ VkImageSubresourceRange VulkanAttachment::getSubresourceRange() const {
             .baseMipLevel = uint32_t(level),
             .levelCount = 1,
             .baseArrayLayer = uint32_t(layer),
-            .layerCount = 1,
+            .layerCount = uint32_t(viewCount) > 0 ? uint32_t(viewCount) : texture->getLayerCount() > 1 ? texture->getLayerCount() : 1,
```

Comment:
It's a neat pick but I don't think the first cast is necessary `viewCount > 0 ?`. I also think that it would be cleaner and more readable if we just moved the whole logic for layerCount comewhere else and then here just do `.layerCount = layerCount`